src/app/pages/send/send-crypto-asset-form/form/stacks-sip10/stacks-sip10-fungible-token-send-form.tsx

```diff
@@ -136,50 +132,63 @@ export function StacksSip10FungibleTokenSendForm({
         sendFormNavigate.toConfirmAndSignStacksSip10Transaction({
           decimals: assetBalance?.balance.decimals,
           name: assetBalance?.asset.name,
-          symbol,
           tx,
         }),
       ledger: () => ledgerNavigate.toConnectAndSignTransactionStep(tx),
     })();
   }
 
+  if (!isString(symbol)) {
+    return <Navigate to={RouteUrls.SendCryptoAsset} />;
+  }
+
   return (
-    <Formik
-      initialValues={initialValues}
-      onSubmit={async (values, formikHelpers) => await previewTransaction(values, formikHelpers)}
-      validationSchema={validationSchema}
-      {...defaultFormikProps}
-    >
-      <NonceSetter>
-        <Form style={{ width: '100%' }}>
-          <AmountField
-            balance={availableTokenBalance}
-            bottomInputOverlay={
-              <SendMaxButton
+    <SendCryptoAssetFormLayout>
+      <Formik
+        initialValues={initialValues}
+        onSubmit={async (values, formikHelpers) => await previewTransaction(values, formikHelpers)}
+        validationSchema={validationSchema}
+        {...defaultSendFormFormikProps}
+      >
+        {props => (
+          <NonceSetter>
+            <Form style={{ width: '100%' }}>
+              <AmountField
                 balance={availableTokenBalance}
-                sendMaxBalance={sendMaxBalance.toString()}
+                bottomInputOverlay={
+                  <SendMaxButton
+                    balance={availableTokenBalance}
+                    sendMaxBalance={sendMaxBalance.toString()}
+                  />
+                }
+              />
+              <FormFieldsLayout>
+                <SelectedAssetField icon={<StxAvatar />} name={symbol} symbol={symbol} />
+                <StacksRecipientField contractId={contractId} />
+                <MemoField />
+              </FormFieldsLayout>
+              <FeesRow fees={stacksFtFees} isSponsored={false} mt="base" />
+              <FormErrors />
+              <PreviewButton
+                isDisabled={
```

Comment:
Didn't we want to remove this?

Reply:
yes, we do. probs added back via bad merge resolution